browser/app/js/components/Browse.js

```diff
@@ -497,338 +508,349 @@ export default class Browse extends React.Component {
+                                 'zmdi-sort-amount-desc': sortSizeOrder,
+                                 'zmdi-sort-amount-asc': !sortSizeOrder
+                               }) } />
+              </div>
+              <div className="objects__item objects__item--modified" onClick={ this.sortObjectsByDate.bind(this) } data-sort="last-modified">
+                Last Modified
+                <i className={ classNames({
+                                 'objects__item__sort': true,
+                                 'zmdi': true,
+                                 'zmdi-sort-amount-desc': sortDateOrder,
+                                 'zmdi-sort-amount-asc': !sortDateOrder
+                               }) } />
               </div>
-              <img className="mh-logo" src={ logo } alt="" />
-            </header>
-            <header className="fe-header">
-              <Path selectPrefix={ this.selectPrefix.bind(this) } />
-              { storageUsageDetails }
-              <ul className="feh-actions">
-                <BrowserUpdate />
-                { loginButton }
-                { browserDropdownButton }
-              </ul>
             </header>
-            <div className="feb-container">
-              <header className="fesl-row" data-type="folder">
-                <div className="fesl-item fesl-item-icon"></div>
-                <div className="fesl-item fesl-item-name" onClick={ this.sortObjectsByName.bind(this) } data-sort="name">
-                  Name
-                  <i className={ classNames({
-                                   'fesli-sort': true,
-                                   'fa': true,
-                                   'fa-sort-alpha-desc': sortNameOrder,
-                                   'fa-sort-alpha-asc': !sortNameOrder
-                                 }) } />
-                </div>
-                <div className="fesl-item fesl-item-size" onClick={ this.sortObjectsBySize.bind(this) } data-sort="size">
-                  Size
-                  <i className={ classNames({
-                                   'fesli-sort': true,
-                                   'fa': true,
-                                   'fa-sort-amount-desc': sortSizeOrder,
-                                   'fa-sort-amount-asc': !sortSizeOrder
-                                 }) } />
-                </div>
-                <div className="fesl-item fesl-item-modified" onClick={ this.sortObjectsByDate.bind(this) } data-sort="last-modified">
-                  Last Modified
-                  <i className={ classNames({
-                                   'fesli-sort': true,
-                                   'fa': true,
-                                   'fa-sort-numeric-desc': sortDateOrder,
-                                   'fa-sort-numeric-asc': !sortDateOrder
-                                 }) } />
+            <div className="objects__container">
+              <Dropzone>
```

Comment:
There's two problems with the Dropzone:
1) There is a slight few pixel padding at the top between the dotted border and the name/size/lastmodified bottom border, and the gap looks odd
2) In firefox, if you drag a file over the dropzone and don't drop it (drag it back out), the dropzone border stays even though you aren't hovering anything

![screen shot 2017-07-25 at 5 04 26 pm](https://user-images.githubusercontent.com/3038254/28598913-a51a790c-715b-11e7-85d4-3af428d3d5f4.png)